dir.props

```diff
@@ -258,7 +258,7 @@
     <NetFxPackageRuntimePath>$(BinDir)pkg\netfx\lib</NetFxPackageRuntimePath>
 
     <!-- We add extra binplacing for the test shared framework until we can get hardlinking with the runtime directory working on all platforms -->
-    <BinPlaceTestSharedFramework Condition="'$(_bc_TargetGroup)' == 'netcoreapp'">true</BinPlaceTestSharedFramework>
+    <BinPlaceTestSharedFramework Condition="'$(_bc_TargetGroup)' == 'netcoreapp' AND '$(BuildTests)'=='true'">true</BinPlaceTestSharedFramework>
```

Comment:
I think it might be worth reversing this condition to say `'$(BuildTests)' != 'false'` because I'm not sure it is always set to true in all cases, but it might be.

Reply:
We already have the same condition here:
https://github.com/dotnet/corefx/blob/bdb916433d3415302ca3931db3b19b7f79b04e44/build.proj#L19

Reply:
That is only a special case to allow building the tests along with the produce from build.sh. It isn't actually ever true when you call build-tests.cmd/sh directly which is what CI is doing.